bootstrap/lib/mix/tasks/nerves/new.ex

```diff
@@ -3,7 +3,11 @@ defmodule Mix.Tasks.Nerves.New do
   import Mix.Generator
 
   @nerves Path.expand("../../../..", __DIR__)
-  @version "0.2"
+
+  @nerves_vsn "0.4.0"
```

Comment:
Our update mechanisms for bootstrap and nerves are more pronounced now, we could lock the others down